Scripts/Runtime/Data/VirtualData/IVirtualData.cs

```diff
@@ -0,0 +1,26 @@
+using Anvil.CSharp.Core;
+using Anvil.Unity.DOTS.Jobs;
+using Unity.Jobs;
+
+namespace Anvil.Unity.DOTS.Data
+{
+    /// <summary>
+    /// Interface for dealing with <see cref="VirtualData{TKey,TInstance}"/> in a generic way without having
+    /// to know the types.
+    /// </summary>
+    public interface IVirtualData : IAnvilDisposable
+    {
+        internal void AddResultDestination(IVirtualData resultData);
+        internal void RemoveResultDestination(IVirtualData resultData);
+        
+        internal JobHandle ConsolidateForFrame(JobHandle dependsOn);
+
+        internal JobHandle AcquireForUpdate();
+        internal void ReleaseForUpdate(JobHandle releaseAccessDependency);
+
+        internal AccessController AccessController
+        {
+            get;
+        }
```

Comment:
In my mind, no because you as the developer will never use this directly. 

If it gets to the point where we do expose it, then yeah docs will be needed

Reply:
Yea just didn't know if developer was potentially a library developer but I'm ok with the rationale.